workflows.yml

```
@@ -358,8 +358,8 @@ software:
         separate_build_job: true
         path: anope
         refs:
-            stable: "2.0.14"
-            release: "2.1.1"
+            stable: "2.0.18"
+            release: "2.1.17"
```

```suggestion
            stable: "2.1.17"
            release: null
```

I don't know why we have both, but there's no reason not to use the newest release as `stable`

---

Heh, doesn't really matter. `stable`'s definition is that it does not change so as long as Anope doesn't force-push the `2.1.17` that's okay. Branch (un)stability is what `devel`/`devel_release` are for.

Let's keep it as is then